launch/2d_slam.launch

```diff
@@ -19,12 +19,13 @@
     <arg name="base_frame" default="base_footprint"/>
     <arg name="odom_frame" default="odom"/>
     <arg name="map_frame" default="map"/>
+    <arg name="image" default="image_rect_raw"/>
```

Comment:
äkki muudaks default väärtuse image_raw'ks, siis pole tarvis argumendina image:=image_raw anda?

Reply:
@Veix123 mis oleks õige default topic siin? kas image_raw või miskit, mis tuleb throttlingu kaudu?

Reply:
arrgh, nvm. kiirustades lugesin. default peabki vist olema image_rect_raw, sest sedasi toimib päris robotil? on nii @mmaao @Veix123 ?
ja ainult gazebos kasutades annad teise image topicu. siin on aga küsimus, kas õnnestuks teha hoopis selliselt, et gazebo käivitamisel saadetakse pilt image_rect_raw rubriiki?

Reply:
Ma võin tegelikult gazebo urdf'is topicu nime ka lihtsalt ära muuta.

Reply:
see tundub hea mõte. teeme nii. sedasi on gazebo setup rohkem päris roboti sarnane!